LanternMobileTestbed/settings.gradle

```diff
@@ -0,0 +1 @@
+include ':app', ':libflashlight'
```

Comment:
Thanks!  I'd been wondering why Intellij kept showing an empty libflashlight project.  I've removed this.